src/test/tests/secretsScan.test.ts

```diff
@@ -0,0 +1,199 @@
+import * as path from 'path';
+
+import { assert } from 'chai';
+import { ConnectionManager } from '../../main/connect/connectionManager';
+import { LogManager } from '../../main/log/logManager';
+import { ScanUtils } from '../../main/utils/scanUtils';
+import { IssuesRootTreeNode } from '../../main/treeDataProviders/issuesTree/issuesRootTreeNode';
+import { createRootTestNode, getTestCodeFileNode } from './utils/treeNodeUtils.test';
+import { ScanResults } from '../../main/types/workspaceIssuesDetails';
+import { AnalyzerUtils } from '../../main/treeDataProviders/utils/analyzerUtils';
+import { getAnalyzerScanResponse } from './utils/utils.test';
+import { FileRegion } from '../../main/scanLogic/scanRunners/analyzerModels';
+import { CodeIssueTreeNode } from '../../main/treeDataProviders/issuesTree/codeFileTree/codeIssueTreeNode';
+import { CodeFileTreeNode } from '../../main/treeDataProviders/issuesTree/codeFileTree/codeFileTreeNode';
+import { SecretsFileIssues, SecretsIssue, SecretsRunner, SecretsScanResponse } from '../../main/scanLogic/scanRunners/secretsScan';
+import { SecretTreeNode } from '../../main/treeDataProviders/issuesTree/codeFileTree/secretsTreeNode';
+
+describe('Secrets Scan Tests', () => {
+    const scanSecrets: string = path.join(__dirname, '..', 'resources', 'secretsScan');
+    let logManager: LogManager = new LogManager().activate();
+
+    describe('Secrets scan fails', () => {
```

Comment:
Let's have more describes for not entitlement.

Reply:
already covered in the tests for the super class